src/proto/uopool/uopool.proto

```diff
@@ -44,6 +44,33 @@ message GetChainIdResponse {
     uint64 chain_id = 2;
 }
 
+message GetSupportedEntryPointsRequest {}
+
+enum GetSupportedEntryPointsResult {
+    GOT_SUPPORTED_ENTRY_POINTS = 0;
+    NOT_GOT_SUPPORTED_ENTRY_POINTS = 1;
+}
+
+message GetSupportedEntryPointsResponse {
+    GetSupportedEntryPointsResult result = 1;
```

Comment:
In practice, it would always `GOT_SUPPORTED_ENTRY_POINTS`. Is it neccessary to add this?